src/TextUI/Command.php

```diff
@@ -505,6 +519,11 @@ protected function handleArguments(array $argv): void
 
                     break;
 
+                case '--stop-on-defect':
```

Comment:
I do not understand why `--stop-on-defect` is needed.

Reply:
Strictly it isn't. I added it as a shorthand for the a combination of `--stop-on-$status` flags. @keradus do you have a more compelling explanation?

Reply:
With current definition of "defect", it's nothing more than a shortcut for putting down `--stop-on-*` multiple times.
It's not mandatory to have, but it's super nice shortcut for fast fail, eg for CLI.
(a lot of users runs all tests in local env, but want to fail on first issue on CI level)